chrome/content/zotero/xpcom/utilities.js

```diff
@@ -455,11 +455,15 @@ Zotero.Utilities = {
 	 *   <ZOTEROBREAK/> => <br/>
 	 *   <ZOTEROHELLIP/> => &#8230;
 	 *
-	 * @param {String} str
+	 * @param {String|Number} str
 	 * @return {String}
 	 */
 	"htmlSpecialChars":function(str) {
-		if (str !== undefined && typeof str != 'string') {
+		if ((str || str == 0) && typeof str != 'string') {
+			if (typeof str != 'number') {
+				Zotero.debug('#htmlSpecialChars: non-string and non-number arguments are deprecated. Update your code', 
```

Comment:
There's not any reason to support number either, though. We should just warn when it's a non-string.